packages/components/chip/package.json

```diff
@@ -28,8 +28,10 @@
   "homepage": "https://sparkui.vercel.app",
   "license": "MIT",
   "dependencies": {
-    "@spark-ui/icon": "latest",
-    "@spark-ui/icons": "latest",
-    "@spark-ui/slot": "latest"
+    "@spark-ui/icon": "^2.0.2",
+    "@spark-ui/icons": "^1.19.5",
+    "@spark-ui/internal-utils": "^2.0.2",
+    "@spark-ui/slot": "^1.6.1",
+    "emulate-tab": "^1.2.1"
```

Comment:
I just ping @kikoruiz :) I am a bit lost with when to use `latest` or not. Has something changed regarding this ?

Reply:
I realise `latest` only works in lerna CI publish step, but locally it does not infer to the current changed package (linking ints changes) and downloads the latest published